src/Jackett.Common/Indexers/RarBG.cs

```diff
@@ -189,23 +181,13 @@ private async Task<IEnumerable<ReleaseInfo>> PerformQueryWithRetry(TorznabQuery
                 case 14: // tmdb not found (see Radarr #7625), thetvdb not found
                 case 20: // no results found
                     if (jsonContent.ContainsKey("rate_limit"))
```

Comment:
only throw ratelimit if `rate_limit` is populated; HTTP is 200; and there are no results found

Reply:
the quoted block is what rarbg indicated for their rate limit handling;  but it seemed when that was implemented in prowlarr they would return a 200 with a rate_limit of 1 and yet not actually be rate limited or something?